api/src/test/java/org/motechproject/nms/api/osgi/UserControllerBundleIT.java

```diff
@@ -142,6 +336,60 @@ public void testNoCallId() throws IOException, InterruptedException {
         httpGet.addHeader("Authorization",
                 "Basic " + new String(Base64.encodeBase64((ADMIN_USERNAME + ":" + ADMIN_PASSWORD).getBytes())));
 
-        SimpleHttpClient.execHttpRequest(httpGet, "{\"failureReason\":\"<callId: Not Present>\"}");
+        assertTrue(SimpleHttpClient.execHttpRequest(httpGet, HttpStatus.SC_BAD_REQUEST,
+                "{\"failureReason\":\"<callId: Not Present>\"}",
+                (String) null, (String) null));
+    }
+
+    @Test
+    @Ignore
```

Comment:
Changing the UserController code to 
```java
    public ResponseUser user(@PathVariable String serviceName, @RequestParam(required = false) String callingNumber,
                             @RequestParam(required = false) String operator,
                             @RequestParam(required = false) String circle,
                             @RequestParam(required = false) String callId)
```
will allow the 'negative' tests to work by allowing passing of empty querystring parameters